articles/access_controls.md

```diff
@@ -7,56 +7,57 @@ layout: article
 Synapse content is subject to two control settings: the `Sharing setting` and `Conditions for Use.` The `Sharing setting` determines _with whom_ content is shared. `Conditions for Use` govern _how_ content can be used. 
 
 ## Sharing setting
-`Sharing setting` determines with whom content (projects, wikis, folders, files, and tables) is shared. Content can be shared (1) **broadly**, with all registered Synapse users, (2) **sparsely,** with a limited set of users (e.g., a team) or (3) can be **private** (i.e., only the creator has access). 
+The `Sharing setting` enables you to invite others to view or collaborate on a project you have created. Projects can be **Public** visible to all or **Private** and visible to selected few. Public projects and contents are shared **broadly** with all registered Synapse users while Private projects and content are shared **sparsely**, with a limited set of designated users (e.g., a team) or not shared at all. **By default, your newly created project is private;** you are the only person that can access it or any content included in it. You can use the `Sharing setting` to enable others to view the content of your projects.
+**Users are responsible for determining the appropriate `Sharing setting` for any content they upload into Synapse.**
 
-**Users are responsible for determining the appropriate `Sharing setting` for any content they upload into Synapse. **
+####Create a team
+If you want to share a project with a specific group of people, you can create a team. Any Synapse user can create a team and invite other Synapse users to join their team. Only after the invited user accepts this invitation is he/she added to the team. Similarly, any Synapse user can request to be added to a team, but will not be added until a team administrator approves the request.
 
```

Comment:
This is optional, You can have someone be automatically added to team without approval